src/diffusers/guiders/classifier_free_guidance.py

```diff
@@ -72,14 +89,17 @@ class ClassifierFreeGuidance(BaseGuidance):
     def __init__(
         self,
         guidance_scale: float = 7.5,
+        distilled_guidance_scale: Optional[float] = None,
         guidance_rescale: float = 0.0,
         use_original_formulation: bool = False,
         start: float = 0.0,
         stop: float = 1.0,
+        enabled: bool = True,
     ):
-        super().__init__(start, stop)
+        super().__init__(start, stop, enabled)
 
         self.guidance_scale = guidance_scale
+        self.distilled_guidance_scale = distilled_guidance_scale
```

Comment:
I am on the fence here but just wanted to get it out there. Should we do a `logger.debug` or something to let the users know that using both `guidance_scale` and `distilled_guidance` could have unexpected effects if the implementations don't support it?